lib/ble_manager.dart

```diff
@@ -37,4 +37,8 @@ abstract class BleManager {
   Future<void> enableRadio({String transactionId});
 
   Future<void> disableRadio({String transactionId});
+
+  Future<List<Peripheral>> knownDevices({List<String> peripheralIdentifiers});
+
+  Future<List<Peripheral>> connectedDevices({List<String> serviceUUIDs});
```

Comment:
I've consulted with @Cierpliwy and this shouldn't be optional and return empty array if an empty array is passed.

Reply:
https://github.com/Polidea/FlutterBleLib/issues/146